site/app/templates/AutogradingStatus.twig

```diff
@@ -0,0 +1,192 @@
+                    Course
+                </th>
+                <th>
+                    Gradeable Id
+                </th>
+                <th>
+                    Interactive
+                </th>
+                <th>
+                    Regrade
+                </th>
+            </tr>
+        </thead>
+        <tbody>
+        {% for sem_course, info in progress["course_info"] %}
+            {% for gradeable, types in info %}
+                <tr>
+                    {% set course_name = sem_course|split("__") %}
+                    <td>{{ course_name[0] }}</td>
+                    <td>{{ course_name[1] }}</td>
+                    <td>{{ gradeable }}</td>
+                    <td>{{ types["interactive"] }}</td>
+                    <td>{{ types["regrade"] }}</td>
+                </tr>
+            {% endfor %}
+        {% endfor %}
+        </tbody>
+    </table>
+    <h2 style="padding-top: 1em;">
+        Machine Statistics
+    </h2>
+    <table class="table table-striped" id="machine-table">
+        <thead>
+            <tr>
+                <th>Machine</th>
+                <th>Semester</th>
+                <th>Course</th>
+                <th>Gradeable Id</th>
+                <th>User Id</th>
+                <th>Elapsed Time</th>
+                <th>Error</th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for machine, worker_info in progress["ongoing_job_info"] %}
+                    {% for info in worker_info %}
+                    <tr>
+                        <td> {{machine}} </td>
+                        <td> {{info["semester"]}} </td>
+                        <td> {{info["course"]}} </td>
+                        <td> {{info["gradeable_id"]}} </td>
+                        <td> {{info["user_id"]}} </td>
+                        <td> {{info["elapsed_time"]}} </td>
+                        <td> {{info["error"]}} </td>
+                    </tr>
+                    {% endfor %}
+            {% endfor %}
+        </tbody>
+    </table>
+</div>
+
+<div class="content table-content custom-scrollbar">
+    <h1>
+        Autograding Monitor
+    </h1>
+    <div id="data" data-machine-num="{{progress["machine_grading_counts"]|keys|length}}" data-capability-num="{{progress["capability_queue_counts"]|keys|length}}" hidden>
+    </div>
+    <table class="table table-striped table-bordered" id="autograding-status-table">
+        <thead>
+            <tr>
+                <th rowspan="3" class="col-md-2 right-boarder">
+                    TIME STAMP
+                </th>
+                <th colspan="2" style="text-align: left;"  class="col-md-4">
+                    INTERATIVE
+                </th>
+                <th colspan="2" style="text-align: left;"  class="col-md-4 right-boarder">
+                    REGRADE
+                </th>
+                <th class="col-md-2 right-boarder" colspan="{{ progress["machine_grading_counts"]|keys|length }}" style="text-align: left;">
```

Comment:
The bootstrap class `.text-left` would do the same thing and remove the need for an inline style.  Same goes for a number of other places in this file.